### ISSUE TITLE
Feature: Different background image for workspaces

### ISSUE THREAD

**Issue by RunicLena** [aleksey-hoffman/sigma-file-manager#286]
## Feature request:
As the title says, allow setting a new background for each workspace, to better differentiate which one you're using

**Comment by aleksey-hoffman** [comment]
@RunicLena good idea. I already started developing workspace customization in v2.0.
I'll see if I can add it in v1.x
